src/app/App.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { ChakraProvider } from '@chakra-ui/react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import Form from '../components/Form/Form.tsx';
+import MainPanel from '../components/MainPanel(Dashboard)/MainPanel';
+import Profile from '../moduels/auth/components/Dashboard/Profile/Profile.js';
+import Fire from '../components/Fire.js';
+function App() {
+  return (
+    <ChakraProvider>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/dash" element={<MainPanel />} />
```

Comment:
`I would rather name it as           <Route path="/dashboard" element={<Dashboard />} />`